### PR TITLE
Fix VSCode debug mode marking tests as failed

### LINKED ISSUE #3191: When debugging using VS Code, the test passes but the report says it is failed in overall reporting -- WebdriverIO 5.0
- **WebdriverIO version:** "webdriverio": "^5.0.3"
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:** sync
 - **Node.js version:** v10.14.2
 - **NPM version:** 6.4.1
 - **Browser name and version:** Version 71.0.3578.98 (Official Build) (64-bit)
 - **Platform name and version:** Mac Mojave 10.14.2
 - **Additional wdio packages used (if applicable):** 
https://github.com/smarigowda/wdio5POC/blob/syncVersionRaisingIssueWithWebdriverIO/package.json

**Config of WebdriverIO**
An example of how you configured WebdriverIO config
https://github.com/smarigowda/wdio5POC/blob/syncVersionRaisingIssueWithWebdriverIO/wdio.conf.js

**Describe the bug**
When debugging the test using VS Code. The test passes but the overall report says it as fail. See attached screenshot.

**To Reproduce**
1. Clone the repository: https://github.com/smarigowda/wdio5POC/tree/syncVersionRaisingIssueWithWebdriverIO

2. npm install

3. Open the repository in VS Code. Open the test spec file. Click on debug icon on the left hand side of VS navigation. Click on green arrow at the top to run/ debug the test. Test should pass and you should see that the spec reporter reports the overall test as fail though the tests are passing.

[Include code or an example repository that can easily be set up]

**Expected behavior**
Overall report should report the status properly.

**Log**
Refer attached screenshot.

### PR DESCRIPTION
**Pull request by mgrybyk**
## Proposed changes

Fix #3191 

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

**VSCode launch.json**
```
    {
      "type": "node",
      "request": "launch",
      "name": "wdio",
      "args": [
        "wdio.conf.js"
      ],
      "cwd": "${workspaceFolder}",
      "autoAttachChildProcesses": true,
      "program": "${workspaceRoot}/node_modules/.bin/wdio",
      "console": "integratedTerminal"
    }
```
NOTE: `"autoAttachChildProcesses": true` is mandatory. If port is not specified then inspect-brk port in wdio.conf.js should not be specified as well.

**wdio.conf.js**
```
    debug: true
```
NOTE: there should be no `execArgv: ['--inspect-brk=127.0.0.1:9229'],` unless port is specified in VSCode launch.json

### CODE REVIEW

File: packages/wdio-local-runner/src/transformStream.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
 import { Transform } from 'stream'
 
+const debuggerMessages = [
+    'Debugger listening on',
+    'Debugger attached',
+    'Waiting for the debugger'
+]
```
Comment (christian-bromann) [inline packages/wdio-local-runner/src/transformStream.js]: Also this should move into https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-local-runner/src/constants.js

**Comment by mgrybyk** [comment]
> The code change looks ok, we should have some tests for it though (https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-local-runner/tests/transformStream.test.js)

tests added, please check if it is enough. Thanks

**Comment by christian-bromann** [comment]
@mgrybyk please sign the CLA

**Comment by mgrybyk** [comment]
@christian-bromann  should I do force push in order to trigger cla update `licence/cla Expected — Waiting for status `  be reported` ? I've singed it already.

**Comment by mgrybyk** [comment]
ok, force push helped

**Comment by kfirbhCS** [comment]
Hi there 

i am using wdio 5.15.6

with vscode 1.42.0

i use the solution mentioned above and remove the port from both places 

tests will pass but VS code wont stop on break point what the reason in that

**Pull request merged**